src/main/java/overlap_crap_fix_locations_later/application_business/OverlapGeneratedTimetableRelayInteractor.java

```diff
@@ -0,0 +1,66 @@
+package overlap_crap_fix_locations_later.application_business;
+
+import entities.Timetable;
+import retrieve_timetable_use_case.application_business.EntityConverter;
+import retrieve_timetable_use_case.application_business.TimetableModel;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Flow;
+
+/**
+ * An interactor whose main purpose is to listen to TimetableGeneratorInteractor, and relay it's data through to
+ * the inputDialog through a presenter.
+ * <p>
+ * I am assuming that this is subscribed to something that gives me a List<Timetable>.
+ */
+public class OverlapGeneratedTimetableRelayInteractor implements Flow.Subscriber<Object> {
```

Comment:
Does this interactor also implement an input boundary of any type?

Reply:
There's no need in this case, since essentially its only job is to listen for data from another interactor and pass it up the food chain to the UI. Note how its only methods come from implementing Flow.Subscriber<Object> - hence the name 'Relay interactor' 😇 . 

You could look at it as not really needing an Input boundary, or you could look at it as having Flow.Subscriber<Object> as its input boundary. But we probably don't need another one on top of that.